readme.md

```diff
@@ -249,6 +251,41 @@ Yields:
 }
 ```
 
+#### `Options`
+
+Configuration (optional).
+
+###### `options.getOnigurumaUrlFetch`
+
+Get a URL to the oniguruma WASM, typically used in browsers (`GetOnigurumaUrl`,
+optional).
+
+###### `options.getOnigurumaUrlFs`
+
+Get a URL to the oniguruma WASM, typically used in Node.js (`GetOnigurumaUrl`,
+optional).
+
+#### `GetOnigurumaUrl`
+
+Function to get a URL to the oniguruma WASM.
+
+> 👉 **Note**: this must currently result in a version 1 URL of
+> `onig.wasm` from `vscode-oniguruma`.
+
+###### Returns
+
+URL object to a WASM binary (`URL` or `Promise<URL>`).
+
+
+###### Example 
+```js
+import {createStarryNight, common} from '@wooorm/starry-night'
+
+const starryNight = await createStarryNight(common, {
+  getOnigurumaUrlFetch: () => new URL(window.location.origin + "onig.wasm")
+})
```

Comment:
```suggestion
const starryNight = await createStarryNight(common, {
  getOnigurumaUrlFetch() {
    return new URL('/onig.wasm', window.location.href)
  }
})
```

a) sticks with code style used in this project (no semicolons, single quotes, no unneeded arrow functions)
b) I think it’s slightly better practise to resolve from the whole current URL, and then in the relative url resolve from it, instead of string concatenation, particularly because I think you’re missing a `/` in the concatenation?

Reply:
yeah you're right, there was a `/` missing, fixed it 63eb1bd